library/src/main/java/com/bumptech/glide/load/resource/bitmap/PositionedCrop.java

```diff
@@ -0,0 +1,83 @@
+package com.bumptech.glide.load.resource.bitmap;
+
+import android.content.Context;
+import android.graphics.Bitmap;
+import android.support.annotation.FloatRange;
+import com.bumptech.glide.load.engine.bitmap_recycle.BitmapPool;
+import java.security.MessageDigest;
+
+/**
+ * Scale the image so that either the width of the image matches the given width and the height of
+ * the image is greater than the given height or vice versa, and then crop the larger dimension to
+ * match the given dimension.
+ *
+ * Using percentages the crop area can be adjusted
+ *
+ * Example usage:
+ *
+ * Crop top-left: new PositionedCrop(context, 0, 0);
```

Comment:
Update the examples to use the constants, especially `0,5f`!
Hmm, actually, after having all the constants, do we really need to say this?

> "Crop top-left: `new PositionedCrop(context, LEFT, TOP)`"

---

Note: this is something I've never understood: if there's a new line between each line then there's no benefit of having the new lines, because they evenly space things out, hence provide no grouping of source code lines. If people prefer bigger line-height than normal, they should set it in their browser/text editor, not in the source code.